src/Error/ExceptionRenderer.php

```diff
@@ -384,9 +398,10 @@ protected function getHttpCode(Throwable $exception): int
      * Generate the response using the controller object.
      *
      * @param string $template The template to render.
+     * @param string $fallbackTemplate The fallback template to render if $template is missing.
      * @return \Cake\Http\Response A response object that can be sent.
      */
-    protected function _outputMessage(string $template): Response
+    protected function _outputMessage(string $template, string $fallbackTemplate = 'error500'): Response
```

Comment:
Provide an explicit default for easier migrations since custom exception renderers might not know what to do for this.